src/pharmpy/plugins/nonmem/records/record.py

```diff
@@ -10,18 +10,13 @@ class Record:
     def __init__(self, content, parser_class):
         self._content = content
         self._parser_class = parser_class
-        self._root = None
+        self._root = self._parser_class(self._content).root
```

Comment:
Is it really necessary to keep a reference to `content` and `parser_class`? These will not be reused.

Reply:
Not if we are parsing directly

Reply:
OK removing those then.